lib/v2/lsnu/jiaowc/tzgg.js

```diff
@@ -0,0 +1,52 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category;
+    const url = category ? `http://jiaowc.lsnu.edu.cn/tzgg/${category}.htm` : 'http://jiaowc.lsnu.edu.cn/tzgg.htm';
```

Comment:
```suggestion
    const url = category ? `https://jiaowc.lsnu.edu.cn/tzgg/${category}.htm` : 'https://jiaowc.lsnu.edu.cn/tzgg.htm';
```
HTTPS is available

Reply:
For unknown reasons, this website cannot be accessed via HTTPS in some network environments, while in others it can be. I have changed all HTTP to HTTPS.